src/drivers/smmstore/store.c

```diff
@@ -47,8 +47,36 @@ _Static_assert(SMM_BLOCK_SIZE <= FMAP_SECTION_SMMSTORE_SIZE,
  * crash/reboot could clear out all variables.
  */
 
+#if CONFIG(DRIVERS_EFI_UPDATE_CAPSULES)
+#define SMMSTORE_CMD_USE_FULL_FLASH 0x80
```

Comment:
Added in https://github.com/Dasharo/coreboot/pull/511/commits/5396e5c0e48880cec75b1046fef421e2889ca887.

While glancing over existing documentation I realized that this approach doesn't pass information about total flash size. Is this something that can be easily detected from edk2? If not, I can pass it in one of unused bytes in `lb_smmstorev2`, but contrary to `num_blocks`, this would have to be compressed to one byte, e.g. by specifying it as MiB instead of number of blocks. This structure is widely used, so it can't be redefined to include more fields.

Reply:
Would be weird to add whole flash size to SMMSTORE information.  It might be one more thing to add to EFI firmware information (https://github.com/Dasharo/coreboot/pull/527#discussion_r1653202476) unless this stuff will end up being passed at build-time.

Reply:
Since coreboot already publishes information and it knows ROM size, adding a new field seems to be the best option.  Inferring information when we can just pass it is weird.